deployments/sherlock-frontend/resources/sherlock-ingress.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: sherlock
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/app-root: "/"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    cert-manager.io/cluster-issuer: "letsencrypt-issuer"
+spec:
+  tls:
+    - hosts:
+      - status.roundtable.lsst.codes
+      secretName: sherlock-ingress-tls
+  rules:
+    - host: status.roundtable.lsst.codes
```

Comment:
If you want to publish to a different subdomain (like taking over `status.lsst.codes`, that's totally fine rather than putting roundtable in the URL. A policy decision more than anything, I guess.

Reply:
Yeah that's what I was thinking, but I wasn't sure if there was something special in here to get under roundtable or if I can put it anywhere.  Is there any best practice on this?  Do I just run it the first time and grab and IP and then set that up in DNS?